newsroom/section_filters/service.py

```diff
@@ -0,0 +1,50 @@
+from superdesk.flask import g
+from superdesk.core.resources import AsyncCacheableService
+
+from newsroom.search.service import query_string
+from newsroom.core.resources.service import NewshubAsyncResourceService
+
+from .model import SectionFilter
+
+
+class SectionFiltersService(NewshubAsyncResourceService[SectionFilter], AsyncCacheableService):
+    resource_name = "section_filters"
+    cache_lookup = {"is_enabled": True}
+
+    async def get_section_filters(self, filter_type) -> list:
+        """Get the list of section filter by filter type
+
+        :param filter_type: Type of filter
+        """
+        section_filters = await self.get_section_filters_dict()
+        return section_filters.get(filter_type) or []
+
+    async def get_section_filters_dict(self) -> dict[str, list]:
```

Comment:
Missing sub-type in return type for `list`

Reply:
do you mean like this? 
```suggestion
    async def get_section_filters_dict(self) -> dict[str, list[dict]]:
```

Reply:
done